### ISSUE TITLE
Pre-work Checklist: Developer: Danny Do

### ISSUE THREAD

**Issue by Adastros** [hackforla/website#3685]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [x] Share your GitHub handle and Gmail address with a merge team member or technical lead so you can be added to the Google Drive
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [x] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Once added to the website-write team:
  - [x] Self Assign this issue (gear in right side panel)
  - [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
  - [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues with increasing complexity in the following order:
  - Good first issue (one per person)
  - Good second issue (one per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by Adastros** [comment]
Provide Update
1. Progress: 28 out of 34 tasks complete. Pending attending meetings to complete issue.
2. Blockers: None
3. Availability:  9 - 11:30 PM Weekdays, Sunday 10AM - 10PM
5. ETA: 13NOV22

**Comment by Adastros** [comment]
How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?

Approximately 8 hours

**Comment by Adastros** [comment]
All tasks for pre-work checklist completed.

**Comment by HackforLABot** [comment]
## Activity Log: Adastros
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- Adastros issue closed by PR 3717: https://github.com/hackforla/website/issues/2904#event-7015838163 at 07/18/2022, 02:37 PM PDT
- Adastros issue closed by PR 3717: https://github.com/hackforla/website/issues/2904#event-7015838163 at 07/18/2022, 02:37 PM PDT
- Adastros opened: https://github.com/hackforla/website/issues/3685 at 10/31/2022, 07:38 PM PDT
- Adastros assigned: https://github.com/hackforla/website/issues/3685 at 11/01/2022, 08:54 PM PDT
- Adastros assigned: https://github.com/hackforla/website/issues/2858#issuecomment-1048436585 at 11/04/2022, 08:39 PM PDT
- Adastros commented on: https://github.com/hackforla/website/issues/2858#issuecomment-1304396830 at 11/04/2022, 09:07 PM PDT
- Adastros opened: https://github.com/hackforla/website/pull/3695 at 11/04/2022, 10:07 PM PDT
- Adastros PR was merged: https://github.com/hackforla/website/pull/3695#event-7755951560 at 11/07/2022, 12:08 PM PST
- Adastros issue closed by PR 3695: https://github.com/hackforla/website/issues/2858#event-7755951720 at 11/07/2022, 12:08 PM PST
- Adastros commented on: https://github.com/hackforla/website/issues/3685#issuecomment-1306627467 at 11/07/2022, 08:44 PM PST
- Adastros assigned: https://github.com/hackforla/website/issues/3207#issuecomment-1145822417 at 11/07/2022, 09:53 PM PST
- Adastros commented on: https://github.com/hackforla/website/issues/3207#issuecomment-1306674318 at 11/07/2022, 09:55 PM PST
- Adastros opened: https://github.com/hackforla/website/pull/3699 at 11/07/2022, 10:52 PM PST
- Adastros commented on: https://github.com/hackforla/website/pull/3703#issuecomment-1309795869 at 11/09/2022, 09:26 PM PST
- Adastros submitted review: https://github.com/hackforla/website/pull/3703#pullrequestreview-1175131208 at 11/09/2022, 10:17 PM PST
- Adastros PR was merged: https://github.com/hackforla/website/pull/3699#event-7786132153 at 11/10/2022, 10:48 AM PST
- Adastros issue closed by PR 3699: https://github.com/hackforla/website/issues/3207#event-7786132312 at 11/10/2022, 10:48 AM PST
- Adastros assigned: https://github.com/hackforla/website/issues/2904#event-7015838163 at 11/10/2022, 10:33 PM PST
- Adastros assigned: https://github.com/hackforla/website/issues/2904#event-7015838163 at 11/10/2022, 10:33 PM PST
- Adastros commented on: https://github.com/hackforla/website/issues/2904#issuecomment-1311289580 at 11/10/2022, 10:39 PM PST
- Adastros commented on: https://github.com/hackforla/website/issues/2904#issuecomment-1311289580 at 11/10/2022, 10:39 PM PST
- Adastros commented on: https://github.com/hackforla/website/issues/3685#issuecomment-1312951717 at 11/13/2022, 06:20 PM PST
- Adastros commented on: https://github.com/hackforla/website/issues/3685#issuecomment-1312953077 at 11/13/2022, 06:23 PM PST
- Adastros closed as completed: https://github.com/hackforla/website/issues/3685#event-7799822610 at 11/13/2022, 06:23 PM PST
- Adastros opened: https://github.com/hackforla/website/pull/3717 at 11/13/2022, 08:54 PM PST
- Adastros commented on: https://github.com/hackforla/website/pull/3717#issuecomment-1314809373 at 11/14/2022, 09:54 PM PST
- Adastros opened: https://github.com/hackforla/website/issues/3720 at 11/15/2022, 12:03 AM PST
- Adastros PR was merged: https://github.com/hackforla/website/pull/3717#event-7829148008 at 11/16/2022, 04:32 PM PST
- Adastros issue closed by PR 3717: https://github.com/hackforla/website/issues/2904#event-7829148120 at 11/16/2022, 04:32 PM PST
- Adastros issue closed by PR 3717: https://github.com/hackforla/website/issues/2904#event-7829148120 at 11/16/2022, 04:32 PM PST
- Adastros commented on: https://github.com/hackforla/website/pull/3728#issuecomment-1318082556 at 11/16/2022, 09:05 PM PST
- Adastros submitted review: https://github.com/hackforla/website/pull/3728#pullrequestreview-1183743795 at 11/16/2022, 10:40 PM PST
- Adastros commented on: https://github.com/hackforla/website/pull/3730#issuecomment-1319545153 at 11/17/2022, 08:55 PM PST
- Adastros submitted review: https://github.com/hackforla/website/pull/3730#pullrequestreview-1185611364 at 11/17/2022, 11:33 PM PST
- Adastros assigned: https://github.com/hackforla/website/issues/3587#issuecomment-1281492060 at 11/18/2022, 10:29 PM PST
- Adastros commented on: https://github.com/hackforla/website/issues/3587#issuecomment-1320816199 at 11/18/2022, 10:31 PM PST
- Adastros opened: https://github.com/hackforla/website/pull/3741 at 11/21/2022, 11:24 PM PST
- Adastros commented on: https://github.com/hackforla/website/pull/3737#issuecomment-1323228621 at 11/21/2022, 11:31 PM PST
- Adastros submitted review: https://github.com/hackforla/website/pull/3737#pullrequestreview-1189555320 at 11/22/2022, 12:09 AM PST
- Adastros commented on: https://github.com/hackforla/website/pull/3736#issuecomment-1323269285 at 11/22/2022, 12:11 AM PST
- Adastros submitted review: https://github.com/hackforla/website/pull/3736#pullrequestreview-1191133282 at 11/22/2022, 11:05 PM PST
- Adastros commented on: https://github.com/hackforla/website/pull/3741#issuecomment-1328197035 at 11/27/2022, 12:40 AM PST
- Adastros unassigned: https://github.com/hackforla/website/issues/2904#issuecomment-1328329813 at 11/27/2022, 12:13 PM PST
- Adastros unassigned: https://github.com/hackforla/website/issues/2904#issuecomment-1328329813 at 11/27/2022, 12:13 PM PST
- Adastros assigned: https://github.com/hackforla/website/issues/3704#issuecomment-1327102396 at 11/27/2022, 12:28 PM PST
- Adastros unassigned: https://github.com/hackforla/website/issues/3704#issuecomment-1328333414 at 11/27/2022, 12:29 PM PST
- Adastros PR was merged: https://github.com/hackforla/website/pull/3741#event-7908411412 at 11/28/2022, 06:39 PM PST
- Adastros issue closed by PR 3741: https://github.com/hackforla/website/issues/3587#event-7908411480 at 11/28/2022, 06:39 PM PST
- Adastros commented on: https://github.com/hackforla/website/pull/3741#issuecomment-1330152281 at 11/28/2022, 10:35 PM PST
- Adastros commented on: https://github.com/hackforla/website/pull/3753#issuecomment-1330153232 at 11/28/2022, 10:37 PM PST
- Adastros commented on: https://github.com/hackforla/website/issues/3720#issuecomment-1330154887 at 11/28/2022, 10:39 PM PST
- Adastros assigned: https://github.com/hackforla/website/issues/2904#issuecomment-1328329813 at 11/28/2022, 10:42 PM PST
- Adastros assigned: https://github.com/hackforla/website/issues/2904#issuecomment-1328329813 at 11/28/2022, 10:42 PM PST
- Adastros commented on: https://github.com/hackforla/website/issues/2904#issuecomment-1330157642 at 11/28/2022, 10:44 PM PST
- Adastros commented on: https://github.com/hackforla/website/issues/2904#issuecomment-1330157642 at 11/28/2022, 10:44 PM PST
- Adastros submitted review: https://github.com/hackforla/website/pull/3753#pullrequestreview-1198706982 at 11/29/2022, 09:33 PM PST
- Adastros commented on: https://github.com/hackforla/website/issues/2904#issuecomment-1333322527 at 11/30/2022, 11:26 PM PST
- Adastros commented on: https://github.com/hackforla/website/issues/2904#issuecomment-1333322527 at 11/30/2022, 11:26 PM PST
- Adastros commented on: https://github.com/hackforla/website/issues/2904#issuecomment-1338858231 at 12/05/2022, 10:34 PM PST
- Adastros commented on: https://github.com/hackforla/website/issues/2904#issuecomment-1338858231 at 12/05/2022, 10:34 PM PST
- Adastros commented on: https://github.com/hackforla/website/issues/2904#issuecomment-1376790694 at 01/09/2023, 10:25 PM PST
- Adastros commented on: https://github.com/hackforla/website/issues/2904#issuecomment-1376790694 at 01/09/2023, 10:25 PM PST
- Adastros opened: https://github.com/hackforla/website/pull/3809 at 01/10/2023, 10:47 PM PST
- Adastros PR was merged: https://github.com/hackforla/website/pull/3809#event-8223594916 at 01/13/2023, 10:58 AM PST
- Adastros issue closed by PR 3717: https://github.com/hackforla/website/issues/2904#event-8223595071 at 01/13/2023, 10:58 AM PST
- Adastros issue closed by PR 3717: https://github.com/hackforla/website/issues/2904#event-8223595071 at 01/13/2023, 10:58 AM PST
- Adastros commented on: https://github.com/hackforla/website/pull/3809#issuecomment-1382965772 at 01/14/2023, 03:45 PM PST
- Adastros commented on: https://github.com/hackforla/website/pull/3810#issuecomment-1382967986 at 01/14/2023, 03:58 PM PST
- Adastros submitted review: https://github.com/hackforla/website/pull/3810#pullrequestreview-1249097256 at 01/14/2023, 06:45 PM PST
- Adastros commented on: https://github.com/hackforla/website/pull/3810#issuecomment-1383267754 at 01/15/2023, 02:17 PM PST
- Adastros submitted review: https://github.com/hackforla/website/pull/3810#pullrequestreview-1250946199 at 01/16/2023, 10:30 PM PST
- Adastros commented on: https://github.com/hackforla/website/pull/3815#issuecomment-1384899484 at 01/16/2023, 10:35 PM PST
- Adastros submitted review: https://github.com/hackforla/website/pull/3815#pullrequestreview-1251023520 at 01/16/2023, 11:29 PM PST
- Adastros commented on: https://github.com/hackforla/website/pull/3838#issuecomment-1396417461 at 01/18/2023, 08:14 PM PST
- Adastros submitted review: https://github.com/hackforla/website/pull/3838#pullrequestreview-1261201204 at 01/18/2023, 10:50 PM PST
- Adastros submitted review: https://github.com/hackforla/website/pull/3838#pullrequestreview-1263038905 at 01/19/2023, 10:52 PM PST
- Adastros commented on: https://github.com/hackforla/website/pull/3864#issuecomment-1401418956 at 01/23/2023, 09:34 PM PST
- Adastros submitted review: https://github.com/hackforla/website/pull/3864#pullrequestreview-1266960610 at 01/23/2023, 11:42 PM PST
- Adastros commented on: https://github.com/hackforla/website/pull/3845#issuecomment-1401509906 at 01/23/2023, 11:51 PM PST
- Adastros commented on: https://github.com/hackforla/website/pull/3864#issuecomment-1401539817 at 01/24/2023, 12:22 AM PST
- Adastros submitted review: https://github.com/hackforla/website/pull/3864#pullrequestreview-1268699923 at 01/24/2023, 08:40 PM PST
- Adastros submitted review: https://github.com/hackforla/website/pull/3845#pullrequestreview-1268799460 at 01/24/2023, 11:11 PM PST
- Adastros submitted review: https://github.com/hackforla/website/pull/3845#pullrequestreview-1276513177 at 01/30/2023, 09:42 PM PST
- Adastros commented on: https://github.com/hackforla/website/pull/3905#issuecomment-1409801675 at 01/30/2023, 09:53 PM PST
- Adastros submitted review: https://github.com/hackforla/website/pull/3905#pullrequestreview-1276593080 at 01/30/2023, 11:07 PM PST
- Adastros submitted review: https://github.com/hackforla/website/pull/3905#pullrequestreview-1278489018 at 01/31/2023, 10:20 PM PST
- Adastros opened: https://github.com/hackforla/website/issues/3973 at 02/13/2023, 11:38 PM PST
- Adastros opened: https://github.com/hackforla/website/issues/3975 at 02/14/2023, 12:00 AM PST
- Adastros commented on: https://github.com/hackforla/website/issues/3973#issuecomment-1429299384 at 02/14/2023, 12:08 AM PST
- Adastros opened: https://github.com/hackforla/website/issues/3976 at 02/14/2023, 12:22 AM PST
- Adastros commented on: https://github.com/hackforla/website/issues/3975#issuecomment-1436182330 at 02/19/2023, 05:44 PM PST
- Adastros commented on: https://github.com/hackforla/website/pull/4011#issuecomment-1437856138 at 02/20/2023, 08:54 PM PST
- Adastros submitted review: https://github.com/hackforla/website/pull/4011#pullrequestreview-1306634001 at 02/20/2023, 09:11 PM PST
- Adastros submitted review: https://github.com/hackforla/website/pull/4020#pullrequestreview-1310690492 at 02/22/2023, 10:53 PM PST
- Adastros submitted review: https://github.com/hackforla/website/pull/4011#pullrequestreview-1310701444 at 02/22/2023, 11:07 PM PST
- Adastros merged: https://github.com/hackforla/website/pull/4011#event-8588424043 at 02/22/2023, 11:08 PM PST
- Adastros closed by PR 4011: https://github.com/hackforla/website/issues/4008#event-8588424157 at 02/22/2023, 11:08 PM PST
- Adastros submitted review: https://github.com/hackforla/website/pull/4038#pullrequestreview-1316923625 at 02/27/2023, 11:20 PM PST
- Adastros commented on: https://github.com/hackforla/website/pull/4048#issuecomment-1447720283 at 02/27/2023, 11:46 PM PST
- Adastros merged: https://github.com/hackforla/website/pull/4048#event-8623813671 at 02/28/2023, 12:03 AM PST
- Adastros closed by PR 4048: https://github.com/hackforla/website/issues/4006#event-8623813933 at 02/28/2023, 12:03 AM PST
- Adastros commented on: https://github.com/hackforla/website/issues/3175#issuecomment-1449247482 at 02/28/2023, 06:43 PM PST
- Adastros submitted review: https://github.com/hackforla/website/pull/4038#pullrequestreview-1319156521 at 02/28/2023, 11:16 PM PST
- Adastros merged: https://github.com/hackforla/website/pull/4038#event-8634771442 at 02/28/2023, 11:17 PM PST
- Adastros closed by PR 4038: https://github.com/hackforla/website/issues/3952#event-8634771589 at 02/28/2023, 11:17 PM PST
- Adastros commented on: https://github.com/hackforla/website/pull/4108#issuecomment-1455490276 at 03/05/2023, 09:43 PM PST
- Adastros commented on: https://github.com/hackforla/website/issues/3973#issuecomment-1455512616 at 03/05/2023, 10:09 PM PST
- Adastros commented on: https://github.com/hackforla/website/issues/3973#issuecomment-1455529648 at 03/05/2023, 10:27 PM PST
- Adastros submitted review: https://github.com/hackforla/website/pull/4108#pullrequestreview-1325631531 at 03/05/2023, 11:22 PM PST
- Adastros commented on: https://github.com/hackforla/website/pull/4108#issuecomment-1455608075 at 03/05/2023, 11:36 PM PST
- Adastros commented on: https://github.com/hackforla/website/pull/4108#issuecomment-1457682510 at 03/06/2023, 11:28 PM PST
- Adastros commented on: https://github.com/hackforla/website/pull/4115#issuecomment-1457690979 at 03/06/2023, 11:36 PM PST
- Adastros submitted review: https://github.com/hackforla/website/pull/4115#pullrequestreview-1327835344 at 03/07/2023, 12:05 AM PST
- Adastros commented on: https://github.com/hackforla/website/pull/4115#issuecomment-1459507458 at 03/07/2023, 09:10 PM PST
- Adastros commented on: https://github.com/hackforla/website/pull/4115#issuecomment-1459598026 at 03/07/2023, 10:09 PM PST
- Adastros closed: https://github.com/hackforla/website/pull/4115#event-8693331373 at 03/07/2023, 10:09 PM PST
- Adastros commented on: https://github.com/hackforla/website/pull/4131#issuecomment-1459600783 at 03/07/2023, 10:13 PM PST
- Adastros commented on: https://github.com/hackforla/website/pull/4121#issuecomment-1459607942 at 03/07/2023, 10:23 PM PST
- Adastros submitted review: https://github.com/hackforla/website/pull/4121#pullrequestreview-1330063876 at 03/07/2023, 11:07 PM PST
- Adastros submitted review: https://github.com/hackforla/website/pull/4121#pullrequestreview-1332093879 at 03/08/2023, 10:36 PM PST
- Adastros merged: https://github.com/hackforla/website/pull/4121#event-8703824045 at 03/08/2023, 10:37 PM PST
- Adastros closed by PR 4121: https://github.com/hackforla/website/issues/4082#event-8703824200 at 03/08/2023, 10:37 PM PST
- Adastros submitted review: https://github.com/hackforla/website/pull/4131#pullrequestreview-1332135120 at 03/08/2023, 11:16 PM PST
- Adastros opened: https://github.com/hackforla/website/issues/4141 at 03/08/2023, 11:33 PM PST
- Adastros commented on: https://github.com/hackforla/website/pull/4131#issuecomment-1463411804 at 03/09/2023, 11:56 PM PST
- Adastros commented on: https://github.com/hackforla/website/issues/3975#issuecomment-1463429993 at 03/10/2023, 12:12 AM PST
- Adastros commented on: https://github.com/hackforla/website/issues/3975#issuecomment-1495446939 at 04/03/2023, 11:58 PM PDT
- Adastros commented on: https://github.com/hackforla/website/issues/3975#issuecomment-1502671608 at 04/10/2023, 09:30 PM PDT
- Adastros opened: https://github.com/hackforla/website/issues/4498 at 04/12/2023, 11:43 PM PDT
- Adastros commented on: https://github.com/hackforla/website/pull/4518#issuecomment-1512453132 at 04/17/2023, 10:25 PM PDT
- Adastros commented on: https://github.com/hackforla/website/pull/4539#issuecomment-1512501343 at 04/17/2023, 11:19 PM PDT
- Adastros submitted review: https://github.com/hackforla/website/pull/4539#pullrequestreview-1389362993 at 04/17/2023, 11:19 PM PDT
- Adastros submitted review: https://github.com/hackforla/website/pull/4511#pullrequestreview-1389398155 at 04/17/2023, 11:45 PM PDT
- Adastros submitted review: https://github.com/hackforla/website/pull/4539#pullrequestreview-1391208305 at 04/18/2023, 08:25 PM PDT
- Adastros merged: https://github.com/hackforla/website/pull/4539#event-9041883127 at 04/18/2023, 08:26 PM PDT
- Adastros closed by PR 4539: https://github.com/hackforla/website/issues/4352#event-9041883220 at 04/18/2023, 08:26 PM PDT
- Adastros submitted review: https://github.com/hackforla/website/pull/4511#pullrequestreview-1391209724 at 04/18/2023, 08:28 PM PDT
- Adastros merged: https://github.com/hackforla/website/pull/4511#event-9041936133 at 04/18/2023, 08:42 PM PDT
- Adastros closed by PR 4511: https://github.com/hackforla/website/issues/4396#event-9041936236 at 04/18/2023, 08:42 PM PDT
- Adastros commented on: https://github.com/hackforla/website/pull/4518#issuecomment-1517320812 at 04/20/2023, 11:17 PM PDT
- Adastros opened: https://github.com/hackforla/website/issues/4551 at 04/21/2023, 12:08 AM PDT
- Adastros opened: https://github.com/hackforla/website/issues/4552 at 04/21/2023, 12:13 AM PDT
- Adastros submitted review: https://github.com/hackforla/website/pull/4560#pullrequestreview-1399175577 at 04/24/2023, 10:54 PM PDT
- Adastros commented on: https://github.com/hackforla/website/pull/4560#issuecomment-1521199923 at 04/24/2023, 11:06 PM PDT
- Adastros commented on: https://github.com/hackforla/website/pull/4554#issuecomment-1521300777 at 04/25/2023, 12:38 AM PDT
- Adastros merged: https://github.com/hackforla/website/pull/4554#event-9089034453 at 04/25/2023, 12:40 AM PDT
- Adastros closed by PR 4554: https://github.com/hackforla/website/issues/4256#event-9089034663 at 04/25/2023, 12:40 AM PDT
- Adastros submitted review: https://github.com/hackforla/website/pull/4560#pullrequestreview-1401054763 at 04/25/2023, 07:26 PM PDT
- Adastros merged: https://github.com/hackforla/website/pull/4560#event-9098240049 at 04/25/2023, 07:27 PM PDT
- Adastros closed by PR 4560: https://github.com/hackforla/website/issues/4381#event-9098240165 at 04/25/2023, 07:27 PM PDT
- Adastros commented on: https://github.com/hackforla/website/issues/4552#issuecomment-1530910009 at 05/01/2023, 10:38 PM PDT
- Adastros assigned: https://github.com/hackforla/website/issues/4530 at 05/02/2023, 12:00 AM PDT
- Adastros commented on: https://github.com/hackforla/website/issues/4530#issuecomment-1530985483 at 05/02/2023, 12:04 AM PDT
- Adastros opened: https://github.com/hackforla/website/issues/4593 at 05/02/2023, 12:35 AM PDT
- Adastros commented on: https://github.com/hackforla/website/issues/4530#issuecomment-1531024173 at 05/02/2023, 12:40 AM PDT
- Adastros commented on: https://github.com/hackforla/website/issues/4530#issuecomment-1532493648 at 05/02/2023, 11:10 PM PDT
- Adastros closed as completed: https://github.com/hackforla/website/issues/4530#event-9151858810 at 05/02/2023, 11:10 PM PDT
- Adastros commented on: https://github.com/hackforla/website/issues/4593#issuecomment-1534089848 at 05/03/2023, 09:47 PM PDT
- Adastros assigned: https://github.com/hackforla/website/issues/4498 at 05/03/2023, 09:49 PM PDT
- Adastros commented on: https://github.com/hackforla/website/pull/4603#issuecomment-1539470291 at 05/08/2023, 11:06 PM PDT
- Adastros submitted review: https://github.com/hackforla/website/pull/4603#pullrequestreview-1418164765 at 05/09/2023, 01:42 AM PDT
- Adastros commented on: https://github.com/hackforla/website/pull/4603#issuecomment-1541229664 at 05/09/2023, 07:49 PM PDT
- Adastros submitted review: https://github.com/hackforla/website/pull/4603#pullrequestreview-1421772270 at 05/10/2023, 10:21 PM PDT
- Adastros commented on: https://github.com/hackforla/website/pull/4643#issuecomment-1543360849 at 05/10/2023, 10:29 PM PDT
- Adastros closed: https://github.com/hackforla/website/pull/4643#event-9218030767 at 05/10/2023, 10:29 PM PDT
- Adastros submitted review: https://github.com/hackforla/website/pull/4603#pullrequestreview-1423761573 at 05/11/2023, 09:39 PM PDT
- Adastros submitted review: https://github.com/hackforla/website/pull/4603#pullrequestreview-1425478285 at 05/13/2023, 07:02 PM PDT
- Adastros merged: https://github.com/hackforla/website/pull/4603#event-9239444298 at 05/13/2023, 07:04 PM PDT
- Adastros closed by PR 4603: https://github.com/hackforla/website/issues/4574#event-9239444309 at 05/13/2023, 07:04 PM PDT
- Adastros opened: https://github.com/hackforla/website/issues/4662 at 05/13/2023, 07:37 PM PDT
- Adastros commented on: https://github.com/hackforla/website/issues/4498#issuecomment-1546793167 at 05/13/2023, 07:42 PM PDT
- Adastros closed as completed: https://github.com/hackforla/website/issues/4498#event-9239474014 at 05/13/2023, 07:42 PM PDT
- Adastros commented on: https://github.com/hackforla/website/pull/4673#issuecomment-1549041761 at 05/15/2023, 11:03 PM PDT
- Adastros closed: https://github.com/hackforla/website/pull/4673#event-9253733630 at 05/15/2023, 11:03 PM PDT
- Adastros commented on: https://github.com/hackforla/website/pull/4664#issuecomment-1549052703 at 05/15/2023, 11:14 PM PDT
- Adastros closed: https://github.com/hackforla/website/pull/4664#event-9253816445 at 05/15/2023, 11:14 PM PDT
- Adastros commented on: https://github.com/hackforla/website/issues/4264#issuecomment-1549054904 at 05/15/2023, 11:16 PM PDT
- Adastros submitted review: https://github.com/hackforla/website/pull/4666#pullrequestreview-1427866816 at 05/16/2023, 12:05 AM PDT
- Adastros submitted review: https://github.com/hackforla/website/pull/4666#pullrequestreview-1429669778 at 05/16/2023, 06:47 PM PDT
- Adastros merged: https://github.com/hackforla/website/pull/4666#event-9263822406 at 05/16/2023, 06:47 PM PDT
- Adastros closed by PR 4666: https://github.com/hackforla/website/issues/4473#event-9263822503 at 05/16/2023, 06:48 PM PDT
- Adastros commented on: https://github.com/hackforla/website/issues/4136#issuecomment-1558599739 at 05/22/2023, 11:18 PM PDT
- Adastros commented on: https://github.com/hackforla/website/pull/4727#issuecomment-1562267451 at 05/24/2023, 09:58 PM PDT
- Adastros closed: https://github.com/hackforla/website/pull/4727#event-9335297452 at 05/24/2023, 09:58 PM PDT
- Adastros commented on: https://github.com/hackforla/website/issues/4670#issuecomment-1562269099 at 05/24/2023, 10:00 PM PDT
- Adastros commented on: https://github.com/hackforla/website/pull/4726#issuecomment-1562364979 at 05/24/2023, 11:44 PM PDT
- Adastros submitted review: https://github.com/hackforla/website/pull/4726#pullrequestreview-1443271892 at 05/25/2023, 12:41 AM PDT
- Adastros submitted review: https://github.com/hackforla/website/pull/4726#pullrequestreview-1443278608 at 05/25/2023, 12:45 AM PDT
- Adastros commented on: https://github.com/hackforla/website/issues/4662#issuecomment-1563898221 at 05/25/2023, 11:59 PM PDT
- Adastros commented on: https://github.com/hackforla/website/issues/4662#issuecomment-1563912965 at 05/26/2023, 12:11 AM PDT
- Adastros commented on: https://github.com/hackforla/website/issues/4662#issuecomment-1565165859 at 05/26/2023, 07:25 PM PDT
- Adastros commented on: https://github.com/hackforla/website/issues/3975#issuecomment-1565195706 at 05/26/2023, 09:06 PM PDT
- Adastros commented on: https://github.com/hackforla/website/issues/4662#issuecomment-1567634442 at 05/29/2023, 06:03 PM PDT
- Adastros closed as completed: https://github.com/hackforla/website/issues/4662#event-9372684446 at 05/29/2023, 06:03 PM PDT
- Adastros merged: https://github.com/hackforla/website/pull/4747#event-9372744344 at 05/29/2023, 06:17 PM PDT
- Adastros closed by PR 4747: https://github.com/hackforla/website/issues/4399#event-9372744434 at 05/29/2023, 06:17 PM PDT
- Adastros commented on: https://github.com/hackforla/website/issues/3975#issuecomment-1569460062 at 05/30/2023, 08:58 PM PDT
- Adastros submitted review: https://github.com/hackforla/website/pull/4726#pullrequestreview-1454557077 at 05/31/2023, 09:57 PM PDT
- Adastros commented on: https://github.com/hackforla/website/pull/4764#issuecomment-1577882380 at 06/05/2023, 09:22 PM PDT
- Adastros submitted review: https://github.com/hackforla/website/pull/4764#pullrequestreview-1469086516 at 06/07/2023, 11:25 PM PDT
- Adastros submitted review: https://github.com/hackforla/website/pull/4764#pullrequestreview-1469104569 at 06/07/2023, 11:38 PM PDT
- Adastros commented on: https://github.com/hackforla/website/pull/4764#issuecomment-1582000245 at 06/08/2023, 12:03 AM PDT
- Adastros commented on: https://github.com/hackforla/website/pull/4764#issuecomment-1582003633 at 06/08/2023, 12:06 AM PDT
- Adastros commented on: https://github.com/hackforla/website/issues/4769#issuecomment-1584116605 at 06/09/2023, 12:38 AM PDT
- Adastros commented on: https://github.com/hackforla/website/issues/4769#issuecomment-1588758037 at 06/13/2023, 01:00 AM PDT
- Adastros commented on: https://github.com/hackforla/website/issues/4769#issuecomment-1588779300 at 06/13/2023, 01:13 AM PDT
- Adastros commented on: https://github.com/hackforla/website/pull/4764#issuecomment-1590553134 at 06/13/2023, 11:28 PM PDT
- Adastros merged: https://github.com/hackforla/website/pull/4789#event-9524167668 at 06/13/2023, 11:35 PM PDT
- Adastros closed by PR 4789: https://github.com/hackforla/website/issues/4758#event-9524167822 at 06/13/2023, 11:35 PM PDT
- Adastros merged: https://github.com/hackforla/website/pull/4764#event-9574970542 at 06/19/2023, 10:02 PM PDT
- Adastros closed by PR 4764: https://github.com/hackforla/website/issues/3985#event-9574970722 at 06/19/2023, 10:02 PM PDT
- Adastros commented on: https://github.com/hackforla/website/issues/4551#issuecomment-1598122157 at 06/19/2023, 10:05 PM PDT
- Adastros commented on: https://github.com/hackforla/website/issues/4136#issuecomment-1598129971 at 06/19/2023, 10:14 PM PDT
- Adastros merged: https://github.com/hackforla/website/pull/4882#event-9646988394 at 06/26/2023, 10:13 PM PDT
- Adastros closed by PR 4882: https://github.com/hackforla/website/issues/4529#event-9646988550 at 06/26/2023, 10:13 PM PDT
- Adastros issue closed as completed: https://github.com/hackforla/website/issues/4769#event-9698192090 at 07/01/2023, 06:04 PM PDT
- Adastros commented on: https://github.com/hackforla/website/issues/4916#issuecomment-1639551543 at 07/17/2023, 11:09 PM PDT
- Adastros merged: https://github.com/hackforla/website/pull/4701#event-9846077295 at 07/17/2023, 11:20 PM PDT
- Adastros closed by PR 4701: https://github.com/hackforla/website/issues/4592#event-9846077490 at 07/17/2023, 11:20 PM PDT
- Adastros assigned: https://github.com/hackforla/website/issues/4769 at 07/17/2023, 11:27 PM PDT
- Adastros commented on: https://github.com/hackforla/website/issues/4769#issuecomment-1639582479 at 07/17/2023, 11:28 PM PDT
- Adastros commented on: https://github.com/hackforla/website/issues/4903#issuecomment-1641401166 at 07/18/2023, 09:41 PM PDT
- Adastros reopened: https://github.com/hackforla/website/issues/4903#issuecomment-1641401166 at 07/18/2023, 09:41 PM PDT
- Adastros commented on: https://github.com/hackforla/website/issues/4769#issuecomment-1645080163 at 07/21/2023, 12:03 AM PDT
- Adastros commented on: https://github.com/hackforla/website/issues/4916#issuecomment-1645093533 at 07/21/2023, 12:16 AM PDT
- Adastros commented on: https://github.com/hackforla/website/issues/4769#issuecomment-1659595835 at 07/31/2023, 10:33 PM PDT
- Adastros merged: https://github.com/hackforla/website/pull/5155#event-10031404045 at 08/07/2023, 11:29 PM PDT
- Adastros closed by PR 5155: https://github.com/hackforla/website/issues/5057#event-10031404196 at 08/07/2023, 11:29 PM PDT
- Adastros opened: https://github.com/hackforla/website/issues/5233 at 08/14/2023, 11:21 PM PDT
- Adastros assigned: https://github.com/hackforla/website/issues/5233 at 08/14/2023, 11:22 PM PDT
- Adastros commented on: https://github.com/hackforla/website/issues/5233#issuecomment-1687498617 at 08/21/2023, 11:06 PM PDT
- Adastros opened: https://github.com/hackforla/website/issues/5282 at 08/21/2023, 11:47 PM PDT
- Adastros commented on: https://github.com/hackforla/website/issues/5233#issuecomment-1687577786 at 08/21/2023, 11:51 PM PDT
- Adastros merged: https://github.com/hackforla/website/pull/5327#event-10216322709 at 08/28/2023, 10:09 PM PDT
- Adastros closed by PR 5327: https://github.com/hackforla/website/issues/5288#event-10216322821 at 08/28/2023, 10:09 PM PDT
- Adastros opened: https://github.com/hackforla/website/issues/5333 at 08/28/2023, 11:23 PM PDT
- Adastros commented on: https://github.com/hackforla/website/issues/5233#issuecomment-1696841831 at 08/28/2023, 11:25 PM PDT
- Adastros submitted review: https://github.com/hackforla/website/pull/5401#pullrequestreview-1612449400 at 09/05/2023, 10:01 PM PDT
- Adastros commented on: https://github.com/hackforla/website/pull/5298#issuecomment-1709551544 at 09/06/2023, 11:29 PM PDT
- Adastros submitted review: https://github.com/hackforla/website/pull/5401#pullrequestreview-1614682720 at 09/06/2023, 11:41 PM PDT
- Adastros merged: https://github.com/hackforla/website/pull/5401#event-10303369442 at 09/06/2023, 11:42 PM PDT
- Adastros closed by PR 5401: https://github.com/hackforla/website/issues/5175#event-10303369559 at 09/06/2023, 11:42 PM PDT
- Adastros submitted review: https://github.com/hackforla/website/pull/5469#pullrequestreview-1621244432 at 09/11/2023, 09:53 PM PDT
- Adastros submitted review: https://github.com/hackforla/website/pull/5298#pullrequestreview-1621518072 at 09/12/2023, 12:32 AM PDT
- Adastros commented on: https://github.com/hackforla/website/pull/5298#issuecomment-1715166791 at 09/12/2023, 12:38 AM PDT
- Adastros merged: https://github.com/hackforla/website/pull/5553#event-10406115189 at 09/18/2023, 11:12 PM PDT
- Adastros closed by PR 5553: https://github.com/hackforla/website/issues/5427#event-10406115403 at 09/18/2023, 11:12 PM PDT
- Adastros submitted review: https://github.com/hackforla/website/pull/5546#pullrequestreview-1632506786 at 09/18/2023, 11:31 PM PDT
- Adastros commented on: https://github.com/hackforla/website/pull/5298#issuecomment-1724909178 at 09/18/2023, 11:35 PM PDT
- Adastros submitted review: https://github.com/hackforla/website/pull/5546#pullrequestreview-1636871528 at 09/20/2023, 09:37 PM PDT
- Adastros submitted review: https://github.com/hackforla/website/pull/5546#pullrequestreview-1643328065 at 09/25/2023, 09:38 PM PDT
- Adastros merged: https://github.com/hackforla/website/pull/5546#event-10469846670 at 09/25/2023, 09:39 PM PDT
- Adastros closed by PR 5546: https://github.com/hackforla/website/issues/5324#event-10469846810 at 09/25/2023, 09:39 PM PDT
- Adastros submitted review: https://github.com/hackforla/website/pull/5298#pullrequestreview-1643512834 at 09/26/2023, 12:14 AM PDT
- Adastros merged: https://github.com/hackforla/website/pull/5298#event-10470990112 at 09/26/2023, 12:16 AM PDT
- Adastros closed by PR 5298: https://github.com/hackforla/website/issues/4903#event-10470990546 at 09/26/2023, 12:16 AM PDT
- Adastros commented on: https://github.com/hackforla/website/issues/4912#issuecomment-1744236013 at 10/02/2023, 10:27 PM PDT
- Adastros closed as completed: https://github.com/hackforla/website/issues/4912#event-10533332045 at 10/02/2023, 10:27 PM PDT
- Adastros commented on: https://github.com/hackforla/website/issues/4769#issuecomment-1744237098 at 10/02/2023, 10:28 PM PDT
- Adastros closed as completed: https://github.com/hackforla/website/issues/4769#event-10533340983 at 10/02/2023, 10:28 PM PDT
- Adastros commented on: https://github.com/hackforla/website/issues/5233#issuecomment-1754536411 at 10/09/2023, 11:58 PM PDT
- Adastros opened: https://github.com/hackforla/website/issues/5730 at 10/16/2023, 10:50 PM PDT
- Adastros commented on: https://github.com/hackforla/website/issues/5233#issuecomment-1765778095 at 10/16/2023, 11:52 PM PDT
- Adastros opened: https://github.com/hackforla/website/issues/5770 at 10/23/2023, 11:43 PM PDT
- Adastros commented on: https://github.com/hackforla/website/issues/5233#issuecomment-1776621003 at 10/23/2023, 11:44 PM PDT
- Adastros merged: https://github.com/hackforla/website/pull/5793#event-10816023533 at 10/30/2023, 11:00 PM PDT
- Adastros closed by PR 5793: https://github.com/hackforla/website/issues/5754#event-10816023774 at 10/30/2023, 11:00 PM PDT
- Adastros submitted review: https://github.com/hackforla/website/pull/5858#pullrequestreview-1716953939 at 11/06/2023, 11:01 PM PST
- Adastros submitted review: https://github.com/hackforla/website/pull/5858#pullrequestreview-1719388697 at 11/07/2023, 09:24 PM PST
- Adastros merged: https://github.com/hackforla/website/pull/5858#event-10895747242 at 11/07/2023, 09:25 PM PST
- Adastros closed by PR 5858: https://github.com/hackforla/website/issues/5790#event-10895747363 at 11/07/2023, 09:25 PM PST
- Adastros commented on: https://github.com/hackforla/website/issues/5333#issuecomment-1822210449 at 11/21/2023, 10:55 PM PST
- Adastros unassigned: https://github.com/hackforla/website/issues/5233 at 01/11/2024, 10:08 PM PST
- Adastros unassigned: https://github.com/hackforla/website/issues/4530 at 06/17/2024, 06:57 PM PDT
